edn/src/edn.rustpeg

```diff
@@ -148,11 +148,11 @@ namespace_separator = "/"
 // TODO: Be more picky here
 // Keywords follow the rules of symbols, except they can (and must) begin with :
 // e.g. :fred or :my/fred. See https://github.com/edn-format/edn#keywords
-symbol_char_initial = [a-z] / [A-Z] / [0-9] / [*!_?$%&=<>]
-symbol_char_subsequent = [a-z] / [A-Z] / [0-9] / [-*!_?$%&=<>]
+symbol_char_initial = [a-z] / [A-Z] / [0-9] / [*!_?$%&=<>.]
```

Comment:
It's hard to have a lot of faith in the spec, because it seems to be taking the tack of defining things imprecisely and then expanding the allowed set when it conflicts with Clojure:

https://github.com/edn-format/edn/issues/62

Probably the best source of truth is the ClojureScript reader package, but I haven't dug into that myself.